src/content/reference/feature-flags.mdx

```diff
@@ -13,18 +13,21 @@ Admin variables are equivalent to defining that variable on every developer's ma
-| OKTETO_COMPOSE_UPDATE_STRATEGY               | Defines the update strategy that the compose must translate (it can be one of: `rolling`/`recreate`/`on-delete`).                                                                                                               | N/A                    |
-| OKTETO_DISABLE_SPINNER                       | If set, disables the spinner rotation.                                                                                                                                                                                           | `false`                |
-| OKTETO_SYNCTHING_VERSION                     | Specifies the Syncthing version the CLI must use.                                                                                                                                                                               | N/A                    |
\ No newline at end of file
+| Environment Variable                             | Description                                                                                                                                                                                                                     | Default Value          |
+|--------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------|
+| OKTETO_SMART_BUILDS_ENABLED                      | Enables the [Smart Builds feature](core/build-service.mdx#smart-builds) that uses a hash of the build context to determine if an image needs to be rebuilt or can be pulled from the registry. This can accelerate build times. | `true`                 |
+| OKTETO_COMPOSE_VOLUME_AFFINITY_ENABLED           | Compose services mounting the same volume will be placed on the same node using Kubernetes's pod affinity.                                                                                                                      | `true`                 |
+| OKTETO_KUBERNETES_TIMEOUT                        | Specifies the timeout while deploying.                                                                                                                                                                                          | `0`                    |
+| OKTETO_FORCE_REMOTE                              | If true, the CLI will run deploy and destroy commands on the cluster instead of running them locally.                                                                                                                           | `false`                |
```

Comment:
can we remove this one? it's gone as part of exposing the deploy on remote behavior from the admin ui